cob_collision_monitor/launch/collision_monitor.launch

```diff
@@ -1,14 +1,17 @@
 <?xml version="1.0"?>
 <launch>
-  <arg name="load_robot_description" default="false"/>
   <arg name="robot" default="$(env ROBOT)"/>
   <arg name="config_path" default="$(find cob_moveit_config)/$(arg robot)/config"/>
+
+  <arg name="load_semantic_description" default="true"/>
```

Comment:
We do not use __this__ launch file in bringup, but [this](https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_bringup/tools/collision_monitor.launch). There, the options have been removed as SRDF is uploaded via cob_moveit_config/upload_config.launch.

I kept both arguments for completeness. I kept load_semantic_description=true to clarify hat collision_monitor requires SRDF in case collision_monitor is started without SRDF being loaded for what ever reason. Keeping load_planning_context doesn't harm

We could also  just remove the whole launch file here...